adapters/gumgum/gumgum.go

```diff
@@ -128,6 +147,26 @@ func preprocess(imp *openrtb.Imp) (string, error) {
 	return zone, nil
 }
 
+func getMediaTypeForImpID(impID string, imps []openrtb.Imp) openrtb_ext.BidType {
+	for _, imp := range imps {
+		if imp.ID == impID && imp.Banner != nil {
+			return openrtb_ext.BidTypeBanner
+		}
+	}
+	return openrtb_ext.BidTypeVideo
+}
+
+func validateVideoParams(video *openrtb.Video) (err error) {
+	if video.W != 0 && video.H != 0 && video.MinDuration != 0 && video.MaxDuration != 0 && video.Placement != 0 && video.Linearity != 0 {
```

Comment:
Do you think it would be easier to read with the logic reversed?
```
if video.W == 0 || video.H == 0 || video.MinDuration == 0 || video.MaxDuration == 0 || video.Placement == 0 || video.Linearity == 0 {
  return &errortypes.BadInput{
    Message: "Invalid or missing video fields",
  }
}
return nil
```

Reply:
Yes, that does read better - I'll update it. Thanks!